vault/token_store.go

```diff
@@ -783,8 +783,8 @@ func NewTokenStore(ctx context.Context, logger log.Logger, core *Core, config *l
 
 		PathsSpecial: &logical.Paths{
 			Root: []string{
-				"revoke-orphan/*",
-				"accessors*",
+				"revoke-orphan",
+				"accessors/",
 			},
```

Comment:
Note to reviewer:

The prior definition of `revoke-orphan/*` is just plain wrong, so it is fixed.

The prior definition of `accessors*` technically works, because it uses a needless wildcard - that change is just for precision.